### DISCUSSION TITLE
make same position equally below between text and image svg

### DISCUSSION THREAD

**Discussion by ariandanim** [tailwindlabs/tailwindcss#10110] [Help]
I find difficult to make my text **My Card** is fall little same position the below with my image svg, 

```
<a href="#">
    <div class="max-w-lg my-10 border border-slate-200 mx-auto p-5 shadow-md hover:bg-[#778da9] rounded-2xl bg-[#e0e1dd] group ">
        <h5 class="font-bold text-blue-800 mb-2 group-hover:text-white text-lg"> <img src="img/folder.svg" class="inline-block">My Card</h5>
        <p class="text-justify group-hover:text-white">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Culpa facere incidunt aperiam deserunt! Esse quas ratione quasi quia, cum sed ipsa voluptatibus pariatur omnis qui, soluta eligendi. Eveniet, libero expedita!</p>
    </div>
</a>
```

this my image result with the code above, I use the black line for the measure position

![My Card](https://user-images.githubusercontent.com/49848117/208252225-299d5174-aed4-4d4e-b839-93be6094b059.png)

**Comment by wongjn** [comment]
You could apply the `vertical-align` property to the `<img>` element or use a flex layout with elements aligned as desired. See example https://play.tailwindcss.com/37dcdJht0X, elements 1, 2 and 3 showcase different `vertical-align` stylings you may be after, whereas elements 4 and 5 showcase flex layout alternatives.

**Reply by ariandanim** [reply]
Tks Dude

**Reply by ProPhycient5** [reply]
Close the discussion by marking above reply as `Answer`.
